src/app/static/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "sass": "gulp sass && gulp inject",
     "js": "webpack",
-    "build": "gulp copy-templates && gulp styles && webpack",
+    "build": "vue-cli-service serve ./src/index.ts",
```

Comment:
`serve` implies that this starts the app, but this task just needs to build a bundle (that the kotlin backend will serve). Could add a separate `serve` task if useful for development.

Reply:
oh yh this was purely for my ease, i can add another command

Reply:
done!